pkg/nexus/upload.go

```diff
@@ -35,5 +35,6 @@ func (c *Client) Upload(repository, group, file string) error {
 	if err != nil {
 		return fmt.Errorf("could not upload component: %w", err)
 	}
+	c.logger().Infof("Successfully uploaded %s to %s", file, link)
```

Comment:
Hmm, do we really want this here on INFO level? What about moving that log output to the task that calls this method?

Reply:
Hm, currently we're calling `Upload()` from `uploadArtifacts()` in the `finish` binary. Wouldn't we kind of break abstractions/information hiding if we move knowledge about the artifact URL there? Should be probably extend the nexus client interface with a method to let the client construct and return the URL?

Reply:
Good point. We could also return the URL from this method to the caller? I think I'd like that.